apps/worker/package.json

```diff
@@ -27,36 +27,39 @@
     "@nestjs/axios": "~2.0.0",
     "@nestjs/common": "9.3.12",
     "@nestjs/core": "9.3.12",
-    "@nestjs/terminus": "9.2.1",
     "@nestjs/platform-express": "9.3.12",
     "@nestjs/swagger": "^6.3.0",
+    "@nestjs/terminus": "9.2.1",
     "@novu/application-generic": "^0.13.0",
-    "@novu/stateless": "^0.13.0",
     "@novu/dal": "^0.13.0",
     "@novu/node": "^0.13.0",
     "@novu/shared": "^0.13.0",
+    "@novu/stateless": "^0.13.0",
     "@novu/testing": "^0.13.0",
     "@sentry/node": "^7.40.0",
     "@sentry/tracing": "^7.40.0",
+    "@types/newrelic": "^9.13.0",
     "axios": "^1.3.3",
     "body-parser": "^1.20.0",
     "bull": "^4.4.0",
     "bullmq": "^3.10.2",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "cross-env": "^7.0.3",
+    "d": "^1.0.1",
```

Comment:
@Cliftonz Another ping regarding this dependency, where is it used?

Reply:
This should be removed now